azurerm/resource_arm_hdinsight_hadoop_cluster_test.go

```diff
@@ -500,6 +563,167 @@ resource "azurerm_hdinsight_hadoop_cluster" "test" {
 `, template, rInt)
 }
 
+func testAccAzureRMHDInsightHadoopCluster_scriptActions(rInt int, rString string, location string) string {
+	template := testAccAzureRMHDInsightHadoopCluster_template(rInt, rString, location)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_storage_blob" "test-install-script-head" {
+	name                   = "test-install-script-head"
+	storage_account_name   = azurerm_storage_account.test.name
+	storage_container_name = azurerm_storage_container.test.name
+	type                   = "Block"
+	source_content         = "echo \"hello world, I'm a head node!'\""
+}
+
+resource "azurerm_storage_blob" "test-install-script-worker" {
+	name                   = "test-install-script-worker"
+	storage_account_name   = azurerm_storage_account.test.name
+	storage_container_name = azurerm_storage_container.test.name
+	type                   = "Block"
+	source_content         = "echo \"hello world, I'm a worker node'!\""
+}
+
+resource "azurerm_hdinsight_hadoop_cluster" "test" {
+  name                = "acctesthdi-%d"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  location            = "${azurerm_resource_group.test.location}"
+  cluster_version     = "3.6"
+  tier                = "Standard"
+
+  component_version {
+    hadoop = "2.7"
+  }
+
+  gateway {
+    enabled  = true
+    username = "acctestusrgw"
+    password = "TerrAform123!"
+  }
+
+  storage_account {
+    storage_container_id = "${azurerm_storage_container.test.id}"
+    storage_account_key  = "${azurerm_storage_account.test.primary_access_key}"
+    is_default           = true
+  }
+
+  roles {
+    head_node {
+      vm_size  = "Standard_D3_v2"
+      username = "acctestusrvm"
+      password = "AccTestvdSC4daf986!"
+
+	  install_script_action {
```

Comment:
This formatting is slightly off in most places `install_script_action` is called